tests/integration.rs

```diff
@@ -2286,14 +2300,15 @@ mod tests {
                 let mut focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
                 let guest = Guest::new(&mut focal);
                 let mut cmd = GuestCommand::new(&guest);
-                cmd.args(&["--cpus", "boot=1"])
+                cmd.args(&["--cpus", "boot=6"])
                     .args(&["--memory", "size=0"])
                     .args(&[
                         "--memory-zone",
                         "size=1G,guest_numa_node=0",
                         "size=2G,guest_numa_node=1",
                         "size=3G,guest_numa_node=2",
                     ])
+                    .args(&["--numa", "id=0,cpus=0-2", "id=1,cpus=3-4", "id=2,cpus=5"])
```

Comment:
Similar question to the other PR: Do we need to add a sanity check whether the `numa` config from user input is valid? E.g. whether `numa id` is continuous or `numa cpus` are mutual exclusive.

Reply:
I answered the numa id on the other PR, I don't think we should verify this one. But for the CPUs, I agree it might be interesting to check if they're mutually exclusive as it doesn't make sense to have some CPUs belonging to multiple NUMA nodes.

Reply:
What happens to CPUs not assigned to a NUMA node?

Reply:
The Linux guest will simply assign them randomly to a NUMA node. But a user who starts using this option should specify the full topology anyway.